src/js/dataMapping/navigation/menuOptions.jsx

```diff
@@ -71,7 +71,8 @@ export const resourceOptions = [
         label: 'Data Model',
         enabled: true,
         url: 'https://fiscal.treasury.gov/data-transparency/DAIMS-current.html',
-        shouldOpenNewTab: true
+        shouldOpenNewTab: true,
+        externalLink: true
```

Comment:
This one should also have `internalDomain: true` (since it is a BFS site)

Reply:
both done, thanks for noticing